colourblind8/colourblind8.py

```diff
@@ -56,3 +56,46 @@ def plot_scatter(self, x, y, labels = None, palette = None,
 
         ax.legend(title = legend_title, bbox_to_anchor=(1.01, 1), loc = "upper left")
         return ax
+    
+    def plot_histogram(self, y, alpha = 1.0, labels = None, palette = None, 
+                title = None, x_lab = None, legend_title = None):
+    
```

Comment:
We are missing docstrings for `plot_histogram` here. I will add that in my pull request

Reply:
Ok great thank you Sabrina